csharp/src/Microsoft.ML.OnnxRuntime/InferenceSession.cs

```diff
@@ -529,6 +529,17 @@ public string EndProfiling()
             }
         }
 
+        /// <summary>
+        /// Return the nanoseconds of profiling's start time
+        /// 
+        public uint GetProfilingStartTimeNs()
+        {
+            uint startTime = 0;
```

Comment:
ulong?

Reply:
Yes, you are right. `uint64_t` in C++ corresponds to `ulong` in C#. Thanks!